package.json

```diff
@@ -10,7 +10,7 @@
     "ng": "ng",
     "start": "ng serve",
     "build-lf-documentation-dev": "ng build",
-    "build-lf-documentation-prod": "ng build --configuration production --output-hashing=none --extractCss=true",
+    "build-lf-documentation-prod": "ng build --configuration production --output-hashing=none",
```

Comment:
is the functionality the same here? I know it forced you to get rid of this, but make sure the behavior didn't change

Reply:
It seems like the behavior didn't change. With extractCss=false, it was used to able to bundle all styles inline into a single JS file, but extractCss=true doesn't do anything different than we already have (have style.css instead of style.js) https://v11.angular.io/cli/build says: extractCSs Deprecated since version 11.0. No longer required to disable CSS extraction for HMR.

Extract CSS from global styles into '.css' files instead of '.js'.